conans/client/build/cmake.py

```diff
@@ -384,7 +384,7 @@ def test(self, args=None, build_dir=None, target=None):
         if isinstance(args, ConanFile):
             raise ConanException(deprecated_conanfile_param_message)
         if not target:
-            target = "RUN_TESTS" if self._compiler == "Visual Studio" else "test"
+            target = "RUN_TESTS" if self.is_multi_configuration else "test"
```

Comment:
I am trying to find info for this, but not clear at all. In theory RUN_TESTS is for IDEs, including XCode, but ``is_multi_configuration`` could return True for XCode too.

Reply:
Yes, from https://www.johnlamp.net/cmake-tutorial-2-ide-integration.html, it seems that RUN_TESTS is also for XCode